features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/util/toHtmlDocument.kt

```diff
@@ -21,8 +21,12 @@ import io.element.android.libraries.matrix.api.timeline.item.event.MessageFormat
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
 
-fun FormattedBody.toHtmlDocument(): Document? {
+fun FormattedBody.toHtmlDocument(senderDisplayNamePrefix: String? = null): Document? {
```

Comment:
Maybe this could be just `prefix` and we could receive `* $senderDisplayName` as the whole prefix?

Reply:
35b7b661f2f3f8415bc2b049b321c9972ba7fca9